src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/80700247/211327058-fc0965f3-c6af-4e21-bc4b-827d89858fdc.png)

You can use * selector only for box-sizing